From a956d17d2aa7fea53bab97715a8f98858e9b4a5c Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 10 May 2019 14:30:36 +0800
Subject: [PATCH] gpgme/lang/python: gpg-error-config should not be used

gpg-error-config was modified by OE to always return an error.
So we want to find an alternative way to retrieve whatever it
is we need.

Upstream-Status: Inappropriate [changes are specific to OE]

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

Rebase to 1.13.0

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 lang/python/setup.py.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lang/python/setup.py.in b/lang/python/setup.py.in
index d6f537e..695c02e 100755
--- a/lang/python/setup.py.in
+++ b/lang/python/setup.py.in
@@ -173,9 +173,8 @@ class BuildExtFirstHack(build):
 
     def _generate_errors_i(self):
 
-        ge_cflags='@GPG_ERROR_CFLAGS@'
         gpg_error_content = self._read_header(
-            'gpg-error.h', ge_cflags.split(' ') if ge_cflags else [])
+            "gpg-error.h", os.environ.get('CFLAGS').split())
 
         filter_re = re.compile(r'GPG_ERR_[^ ]* =')
         rewrite_re = re.compile(r' *(.*) = .*')
